source/getting-started/advanced-tutorials/fpga-ip-core/index.md

```diff
@@ -0,0 +1,611 @@
+# Tutorial: Custom FPGA IP Core
+
+- **Goal:** Run custom Verilog code in the FPGA.
+- **Complexity:** 5 / 5
+- **Estimated Time:** 90 min
+
+This tutorial goes over the procedure to create and add a custom FPGA IP core in Vivado.
+Step by step instructions are given which walk the user through each part.
+
+## Tutorial Requirements
+
+1. Working AMDC hardware for testing
+2. Completion of the [VSI tutorial](/getting-started/tutorials/vsi/index.md)
+3. Review of the `v1.0` firmware [system architecture documentation](/firmware/arch/system)
+4. Working knowledge of digital logic and Verilog
+    - UW-Madison courses ECE 551/552
+    - Online tutorials, e.g., [1](https://nandland.com/fpga-101/), [2](https://fpgatutorial.com/verilog/), [3](http://www.asic-world.com/verilog/veritut.html), etc
+
+## Background
+
+The AMDC processor is the Xilinx Zynq-7000 SoC---see its [Technical Reference Manual](https://docs.xilinx.com/v/u/en-US/ug585-Zynq-7000-TRM).
+It includes a "Processing System" (PS) and "Programmable Logic" (PL). In the other AMDC docs besides this tutorial, the PS is referred to as the DSP, and PL means FPGA.
+Keep in mind that this tutorial is applicable to any system built on top of Xilinx's system-on-chip (PL + PS) architecture family, not just the AMDC/PicoZed/Zynq-7000 combination.
```

Comment:
Good point, I redirected the text to discuss the PS/PL as an aside to help the reader understand the Xilinx docs, and not a main point. I also changed the rest of this tutorial to use DSP and FPGA instead of PS and PL.